cypress/integration/brewery/ScenarioParametersValidation.spec.js

```diff
@@ -89,4 +89,47 @@ describe('scenario parameters inputs validation', () => {
       });
     }
   );
+  it(
+    'checks error message for date input when given value exceeds minimum and maximum value' +
+      'or expected format is not respected',
+    () => {
+      ScenarioParameters.expandParametersAccordion();
+
+      BreweryParameters.switchToAdditionalParametersTab();
+
+      ScenarioParameters.getInputValue(BreweryParameters.getStartDateInput()).as('start_date');
+      ScenarioParameters.getInputValue(BreweryParameters.getAdditionalDateInput()).as('additional_date');
+
+      BreweryParameters.getStartDateInput().clear();
+      BreweryParameters.getStartDateHelperText().should('be.visible').contains('required');
+      BreweryParameters.getStartDateInput().type('22/22/2222');
+      BreweryParameters.getStartDateHelperText().should('be.visible').contains('format');
+      BreweryParameters.getStartDateInput().clear().type('05/18/2099');
+      BreweryParameters.getStartDateHelperText().should('not.exist');
+
+      BreweryParameters.getAdditionalDateInput().clear().type('05/05/2018');
+      BreweryParameters.getAdditionalDateHelperText().should('exist').contains('Minimum date');
+      BreweryParameters.getAdditionalDateInput().clear().type('05/05/2024');
+      BreweryParameters.getAdditionalDateHelperText().should('exist').contains('Maximum date');
```

Comment:
Could we also add checks on the exact days of `minDate` and `maxDate`, the day just before `minDate`, and the one just after `maxDate`?